File: codechef/hs08test.c
/*
    codechef.com - Practice problem - Easy
    ATM - http://www.codechef.com/problems/HS08TEST
    Language - C
    Juliana Correa
*/

#include <stdio.h>
#define TAX 0.5
     
int main() {

    int wd;
    float bal;
     
    scanf ("%d %f", &wd, &bal);
    float amount = wd + TAX;
     
    if ((wd % 5 == 0) && (amount <= bal))
        bal -= amount;
    
    printf ("%.02f\n", bal);

    return 0;
} 

File: codechef/hs08test.f90
!   codechef.com - Practice problem - Easy
!    ATM - http://www.codechef.com/problems/HS08TEST
!    Language - Fortran
!    Juliana Correa

program atm
implicit none

   integer :: wd
   real    :: bal, amount, tax = 0.5

   read *, wd, bal
   amount = wd + tax
   
   if ((mod(wd, 5) == 0) .and. (amount <= bal)) then
      bal = bal - amount
   end if      
   
   write (*, fmt='(f0.2)') bal

end program atm
